spring-cloud-gcp-docs/src/main/asciidoc/trace.adoc

```diff
@@ -53,13 +53,15 @@ messages exceed its buffer size or have been buffered for longer than its schedu
 
 There are several parameters you can use to tune the Stackdriver Trace adapter. All configurations are optional:
 
-
 |===
 | Property name | Description | Default value
 | `spring.cloud.gcp.trace.enabled` | Auto-configure Spring Cloud Sleuth to send traces to Stackdriver Trace. | `true`
 | `spring.cloud.gcp.trace.executor-threads` | Number of threads to use by the underlying gRPC channel to send the trace request to Stackdriver. | `4`
 | `spring.cloud.gcp.trace.buffer-size-bytes` | Buffer size in bytes. Traces will be flushed to Stackdriver when buffered trace messages exceed this size. | 1% of Runtime.totalMemory()
 | `spring.cloud.gcp.trace.scheduled-delay-seconds` | Buffered trace messages will be flushed to Stackdriver when buffered longer than scheduled delays (in seconds) even if the buffered message size didn't exceed the buffer size. | `10`
+| `spring.cloud.gcp.trace.project-id` | Overrides the project ID from the <<spring-cloud-gcp-core, Spring Cloud GCP Core Module>> |
```

Comment:
Should we remove the space here after comma before ` Spring Cloud` in the link as well?
Maybe we don't need to worry at all?
Would you mind to share screen shot how the final doc looks here?

Thanks

Reply:
![screenshot from 2017-10-24 11 30 45](https://user-images.githubusercontent.com/3290856/31952591-f40abf14-b8ae-11e7-8580-e0991c1be99e.png)